aptos-move/block-executor/src/view.rs

```diff
@@ -1293,6 +1315,9 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
                                 return Some(Ok((key.clone(), (metadata_op, group_size.get()))));
                             }
                         } else {
+                            // TODO: `get_group_size` can fail on group tag serialization. Do
```

Comment:
for "`get_group_size` can fail on group tag serialization" you mean that it can actually fail in prod, based on user input, not based on bug in the code?

if that's the case, that shouldn't be code_invariant_error - because those are not allowed to be committed on chain

Reply:
Yes, we probably need PanicOr in the trait in this case